packages/frontend/src/components/Explorer/SavedChartsHeader/index.tsx

```diff
@@ -486,14 +487,23 @@ const SavedChartsHeader: FC = () => {
                                         />
                                     ) : null}
                                     <Divider />
-                                    <MenuItem2
-                                        icon={<IconTrash />}
-                                        text="Delete"
-                                        intent="danger"
-                                        onClick={() =>
-                                            setIsDeleteDialogOpen(true)
-                                        }
-                                    />
+                                    <Tooltip
+                                        disabled={!getIsEditingDashboardChart()}
+                                        position="bottom"
+                                        label="This chart can be deleted from its dashboard"
```

Comment:
"This chart can only be deleted from the dashboard it was created from" ? IDK, what do you think?

Reply:
I think it needs to be shorter 😄. But I don't have a much better suggestion. I will see what Priyanka thinks when she has time.

Reply:
Cool, this LGTM, I will let @ZeRego review the rest if that's ok 👍